packages/rest/src/lib/REST.ts

```diff
@@ -194,6 +202,11 @@ export interface APIRequest {
 	route: string;
 }
 
+export interface ResponseLike
+	extends Pick<Response, 'arrayBuffer' | 'bodyUsed' | 'headers' | 'json' | 'ok' | 'status' | 'text'> {
```

Comment:
```suggestion
	extends Pick<Response, 'arrayBuffer' | 'headers' | 'json' | 'ok' | 'status' | 'text'> {
```

bodyUsed isn't used anywhere.

Reply:
I decided to hoist it since it felt like it  might be useful to consumers that use .raw or the Response event